modules/rte_mis_gin/rte_mis_gin.module

```diff
@@ -5,6 +5,28 @@
  * Contains rte_mis_gin.module.
  */
 
+// libraries attached to differ pages
+
+function rte_mis_gin_page_attachments(array &$attachments) {
+  // Attach the library to all pages.
+  $attachments['#attached']['library'][] = 'rte_mis_gin/rte_mis_global';
+}
+
+function rte_mis_gin_preprocess_page__user_login(&$variables) {
+  // Implements hook_user_login().
+  $variables['#attached']['library'][] = 'rte_mis_gin/rte_mis_login';
+}
+
+function rte_mis_gin_preprocess_page__user_password(&$variables) {
+  // Implements hook_user_password().
+  $variables['#attached']['library'][] = 'rte_mis_gin/rte_mis_password';
+}
+
+function rte_mis_gin_preprocess_page__user_register(&$variables) {
+  // Implements hook_user_register().
+  $variables['#attached']['library'][] = 'rte_mis_gin/rte_mis_register';
+}
```

Comment:
Looks like this is wrong, Please ref this: https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Render%21theme.api.php/function/hook_preprocess_HOOK/10